docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -0,0 +1,78 @@
+# CheckBox
+CheckBox is a common component and describes what action will occur when you select it.  
+A CheckBox can only contain an icon, and can be created using style.
+
+![CheckBox1](./media/CheckBox1.png)
+
+## Create with Style
+
+To create a CheckBox applying style, follow these steps:
+
+1. Create CheckBox using the default constructor:
+
+    ```cs
+    CheckBox utilityCheckBox = new CheckBox();
+    ```
+
+2. Apply style to the RadioButton:
```

Comment:
Please change RadioButton to CheckBox:

2. Apply style to CheckBox:

Reply:
fixed